README.asciidoc

```diff
@@ -0,0 +1,323 @@
+= %placeholder% 
+
+This is where we're going to write down what we've done so far.
+
+== Members
+
+[horizontal]
+Timo Jairus:: programming
+Artur Salumäe:: electronics
+Jens Olaf Kukk:: mechanics
+
+== Repositories
+* Mechanics: https://a360.co/3SYDOoi
```

Comment:
Mechanics link is to test thrower.
Link to new robot assembly should be added.

Reply:
OK